htsget-config/docs/examples/auth.toml

```diff
@@ -12,16 +12,17 @@ jwks_url = "https://www.example.com/htsget"
 ## Alternatively use public key-based validation
 #public_key = "/path/to/jwt-public-key.pem"
 
-# Trusted authorization URLs
-trusted_authorization_urls = ["https://www.example.com/authorize"]
-# Optionally fetch the authorization url from the path in the JWT. Only
-# issues the request to the URL if it's also in `trusted_authorization_urls`.
-authorization_path = "$.authorization_url"
-
 validate_audience = ["htsget-server"]
 validate_issuer = ["https://www.example.com"]
 validate_subject = "htsget"
 
+# Authorization URLs, if using an authorization service.
+authorization_url = "https://www.example.com/authorize"
+# Forward the authorization header to the auth server.
+passthrough_auth = true
+## Any headers to forward
+#forward_headers = ["Content-Type"]
+
```

Comment:
these forwarded in a JSON packet? Or forwarded as actual headers in the auth url get?

Reply:
Yes, it gets forwarded as a header with `Htsget-Context-` prefix. In the settings, the name of the header also has to be set (minus the prefix) for the JSON path that retrieves it. Also, the JSON path has to resolve to a single JSON value, it can't be an array or object.